pkg/apis/acid.zalan.do/v1/postgresql_type.go

```diff
@@ -220,6 +220,7 @@ type Sidecar struct {
 	DockerImage string             `json:"image,omitempty"`
 	Ports       []v1.ContainerPort `json:"ports,omitempty"`
 	Env         []v1.EnvVar        `json:"env,omitempty"`
+	Command     []string           `json:"command,omitempty" protobuf:"bytes,3,rep,name=command"`
```

Comment:
why is `protobuf:"bytes,3,rep,name=command"` necessary? We have it nowhere else. Is it similar to filter inputs using pattern in the CRD schema?

Reply:
apologies just seeing this comment. I can remove that part